README.md

```diff
@@ -0,0 +1,36 @@
+# EmptyChronicle
+
+EmptyChonicle (or 0c) is simple node for NineChronicles providing http api about block, state, etc.
+
+## Architecture
+
+I recommend reading this section and follow this guide if you want to contribute to this proejct.
+
+```
+EmptyChronicle
+├── Domain
+│   ├── Model
+│   │   ├── Entity
+│   │   └── Repository (** Interface)
+│   └── Service (** Domain Service)
+├── Application
+│   ├── Application (** Application Service)
+│   └── ApplicationInjectionExtensions.cs
+│       You should add applications as dependancy into AIE.cs
+├── Controller
+│   └── Controller (** Application Service)
```

Comment:
I am not sure if it is a typo from copying the above Application section. Could you confirm whether it is a typo?

Reply:
By internal chatting, I checked it is a typo. We decided to remove the part.